.github/workflows/publish.yml

```diff
@@ -0,0 +1,21 @@
+name: Publish to NPM
+on:
+  release:
+    types: [published]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Check out the repo'
+        uses: actions/checkout@v3
+      - name: 'Install Node.js'
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18.x'
```

Comment:
considered using `.nvmrc` for single source of truth with node version?
https://github.com/BirthdayResearch/quantum/blob/main/.github/workflows/ci.yml#L51